frontstage/templates/surveys/surveys-share/survey-select.html

```diff
@@ -70,22 +70,25 @@
 {% endif %}
 
 <h1>Which surveys do you want to share?</h1>
-{% set checkboxData = [] %}
 <form action="" method="post">
     <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+    {% for key in share_dict %}
```

Comment:
Can key be changed to better describe what it is?

Reply:
👍